src/redux/configureStore.js

```diff
@@ -1,5 +1,6 @@
 import { combineReducers, createStore, applyMiddleware } from 'redux';
 import logger from 'redux-logger';
+import thunk from 'redux-thunk';
 import booksReducer from './books/books';
```

Comment:
- [ ] Hi @melaniesigrid, please let's make sure that after refactoring our app to use the Bookstore API, the AddBook and Remove book feature should work the same. When I try to post a book to the server, I get a console error. Our book data is not persisted correctly to the server. You may consider looking critically into the importation `import booksReducer from './books/books';`

Reply:
Thank you for your review! I have corrected the issues sending and receiving from the API.